README.md

```diff
@@ -24,16 +24,19 @@ Features
 Supported HTTP libraries
 ------------------------
 
-* Net::HTTP and libraries based on Net::HTTP (i.e RightHttpConnection, REST Client, HTTParty)
-* HTTPClient
-* Patron
-* EM-HTTP-Request
+* Async::HTTP::Client
 * Curb (currently only Curb::Easy)
-* Typhoeus (currently only Typhoeus::Hydra)
+* EM-HTTP-Request
 * Excon
-* HTTP Gem
+* HTTPClient
+* [HTTP (The Gem! also known as http.rb)](https://github.com/httprb/http)
 * Manticore
-* Async::HTTP::Client
+* Net::HTTP and libraries based on Net::HTTP:
```

Comment:
@BKSpurgeon WebMock supports these and other clients based on Net::HTTP. This change suggests it only supports the listed 3 libs. I suggest adding `e.g.` or `like`